### PR TITLE
Added Layout & Keymap

### PR DESCRIPTION
**Pull request by mrsendyyk**
## Description
@ai03-2725 Soyuz:
1. Added `LAYOUT_5x4`
2. Added `keymap.c`
3. Added `readme.md`
4. Update `rules.mk`

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/soyuz/soyuz.h [hunk 1]
```diff
@@ -39,3 +39,31 @@
...
+ * ├───┼───┼───┤23 │
+ * │20 │21 │22 │   │
+ * ├───┼───┼───┼───┤
+ * │30 │31 │32 │   │
+ * ├───┴───┼───┤43 │
+ * │41     │42 │   │
+ * └───────┴───┴───┘
+*/
+
+#define LAYOUT_5x4( \
```
Comment (zvecr) [inline keyboards/ai03/soyuz/soyuz.h:57]:
```suggestion
#define LAYOUT_numpad_5x4( \
```

File: keyboards/ai03/soyuz/soyuz.h [hunk 2]
```diff
@@ -39,3 +39,31 @@
 	{ K30, K31, K32, K33 }, \
 	{ K40, K41, K42, K43 }  \
 }
+
+/* LAYOUT_5x4
```
Comment (zvecr) [inline keyboards/ai03/soyuz/soyuz.h:43]:
```suggestion
/*
```

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   * |--------------|+   |
+   * |4   |5   |6   |    |
+   * |-------------------+
+   * |1   | 2  |3   |    |
+   * |--------------|Entr|
+   * |0        |.   |    |
+   * `-------------------'
+   */
+
+  [0] = LAYOUT_5x4(KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
```
Comment (zvecr) [inline keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:19]:
```suggestion
  [0] = LAYOUT_numpad_5x4(KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
```

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+};
+
+void matrix_scan_user(void) {
+
+};
+
+void led_set_user(uint8_t usb_led) {
+
+}
```
Comment (zvecr) [inline keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:40]:
```suggestion
```

**Comment by zvecr** [comment]
<https://github.com/qmk/qmk_firmware/blob/7c7078faf36b9d302186f8dd4b7152ab159df050/keyboards/ai03/soyuz/rules.mk#L34>

Should also be updated to `LAYOUTS = ortho_5x4 numpad_5x4`

File: keyboards/ai03/soyuz/soyuz.h (hunk 1 above)
Comment (mrsendyyk) [reply keyboards/ai03/soyuz/soyuz.h]: Done

File: keyboards/ai03/soyuz/soyuz.h (hunk 2 above)
Comment (mrsendyyk) [reply keyboards/ai03/soyuz/soyuz.h:43]: Done

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c (hunk 3 above)
Comment (mrsendyyk) [reply keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:19]: Done

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c (hunk 4 above)
Comment (mrsendyyk) [reply keyboards/ai03/soyuz/keymaps/mrsendyyk/keymap.c:40]: Done

**Comment by mrsendyyk** [comment]
> 
> 
> https://github.com/qmk/qmk_firmware/blob/7c7078faf36b9d302186f8dd4b7152ab159df050/keyboards/ai03/soyuz/rules.mk#L34
> 
> Should also be updated to `LAYOUTS = ortho_5x4 numpad_5x4`

Done @zvecr

File: keyboards/ai03/soyuz/keymaps/mrsendyyk/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+# [Sendy YK](https://mr.sendyyk.com)'s Keymap
```
Comment (fauxpark) [inline keyboards/ai03/soyuz/keymaps/mrsendyyk/README.md:1]: Please rename this file to lowercase.

Comment (mrsendyyk) [reply keyboards/ai03/soyuz/keymaps/mrsendyyk/README.md]: Done

**Comment by zvecr** [comment]
Thanks!

**Comment by mrsendyyk** [comment]
Thank you.

**Comment by mrsendyyk** [comment]
[#794](https://github.com/qmk/qmk_configurator/issues/794) on  [qmk/qmk_configurator](https://github.com/qmk/qmk_configurator)

**Pull request merged**